src/components/Menu/index.tsx

```diff
@@ -258,6 +278,29 @@ export default function Menu() {
     mixpanelHandler(MIXPANEL_TYPE.MENU_PREFERENCE_CLICK, { menu: name })
   }
 
+  const wrapperNode = useRef<HTMLDivElement>(null)
+  const [showScroll, setShowScroll] = useState<boolean>(false)
+
+  useEffect(() => {
+    const wrapper = wrapperNode.current
+    if (wrapper) {
+      const abortController = new AbortController()
```

Comment:
const  abort = useRef(false)

Reply:
How can you implement this? Can you provide full code? Im pretty sure this wont work.

Reply:
This part looks weird to me.

Once onScroll is removed from listeners in cleanUp function, it won't be called again. Then, a new onScroll is created and the old onScroll from the last render will be gone 🤔